src/vs/workbench/browser/actions/media/screencast.css

```diff
@@ -21,15 +21,13 @@
 	position: absolute;
 	background-color: rgba(0, 0, 0 ,0.5);
 	width: 100%;
-	height: 100px;
 	bottom: 20%;
```

Comment:
Could you also please remove `20%` since this is also configurable and it is being set through code.